diagnostics/src/main.rs

```diff
@@ -24,24 +25,30 @@ enum Commands {
 
 #[derive(Debug, Args)]
 struct CommonOpts {
+    /// Kafka message broker, should have format `host:port`, e.g. `localhost:19092`
     #[clap(long)]
     broker: String,
 
+    /// Optional Kafka username
```

Comment:
Worth mentioning the requirement to specify either username and password or neither.